man/dcast.data.table.Rd

```diff
@@ -17,7 +17,7 @@
 \arguments{
   \item{data}{ A \code{data.table}.}
   \item{formula}{A formula of the form LHS ~ RHS to cast, see Details.}
-  \item{fun.aggregate}{Should the data be aggregated before casting? If the formula doesn't identify a single observation for each cell, then aggregation defaults to \code{length} with a 'dt_missing_fun_aggregate_warning'.
+  \item{fun.aggregate}{Should the data be aggregated before casting? If the formula doesn't identify a single observation for each cell, then aggregation defaults to \code{length} warning of class 'dt_missing_fun_aggregate_warning'.
```

Comment:
sorry i didn't fully understand what you want to say with "with a warning" so i made it "with a warning of class" because i think telling people that 'dt_missing_fun_aggregate_warning' is class of the warning is better than just writing that it is the warning, but if it feels wrong to you we can try something else as well :)